<launch>
	<node ns="indoor/laser_utils/viz" name="viz" pkg="laser_node" type="laser_utils_viz"  output="screen" args="/plate_top_link"/>
</launch>
